static/app/views/releases/list/releasesRequest.tsx

```diff
@@ -29,13 +29,7 @@ import {getCrashFreePercent} from 'sentry/views/releases/utils';
 import {ReleasesDisplayOption} from './releasesDisplayOptions';
 
 function omitIgnoredProps(props: Props) {
-  return omit(props, [
-    'api',
-    'organization',
-    'children',
-    'selection.datetime.utc',
-    'location',
```

Comment:
Is this intentional?

Reply:
Yep, removing `location` from the ignored props allows changes to the filtering to reload the health stats like crash free rate. Before it'd remount the whole thing so it'd reload anyway